src/components/todo-item/TodoItem.jsx

```diff
@@ -1,9 +1,6 @@
 import style from "./TodoItem.module.css";
 
-export function TodoItem(props) {
-  const {
-    todo: { id, content, checked },
-  } = props;
+export function TodoItem({ todo: { id, content, checked }, deleteTodo }) {
```

Comment:
Please follow [this article](https://spencerpauly.com/tech/react-naming-conventions-best-practices/) to adjust your prop names.